### PR TITLE
Use octoheads project for generating avatars

### LINKED ISSUE #244: Extract octopus avatars generator to a separate library
We can fork and modify https://github.com/RobertBroersma/bigheads project

### PR DESCRIPTION
**Pull request by pastpatryk**
Use custom [mavend/octoheads](https://github.com/mavend/octoheads) package for octopus avatars generation

Closes #244 

Screenshot 2021-04-06 at 23 07 35
Screenshot 2021-04-07 at 09 28 39

### CODE REVIEW

File: src/components/user/Avatar/index.js [hunk 1]
```diff
@@ -1,106 +1,36 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import { bodies, DefaultBody } from "./parts/bodies";
```
Comment (glowacki-dev) [inline src/components/user/Avatar/index.js:4]: Those are not used anymore and can be removed

File: src/components/user/Avatar/index.js [hunk 2]
```diff
@@ -1,106 +1,36 @@
...
-      >
-        <g style={{ fill: color, stroke: color, ...bodyStyle }}>
-          <Body />
-        </g>
-        <Eyes />
-        <Features />
-        <Glasses />
-        <Border />
-      </svg>
+    <div style={styles} {...props}>
```
Comment (glowacki-dev) [inline src/components/user/Avatar/index.js:32]: Do we ever pass any props here?

**Review by glowacki-dev — CHANGES_REQUESTED** [review]

File: src/components/user/Avatar/index.js (hunk 2 above)
Comment (pastpatryk) [reply src/components/user/Avatar/index.js:32]: no, but we may pass e.g. `className` if we decide to style it somewhere using CSS modules

File: src/components/user/Avatar/index.js (hunk 1 above)
Comment (pastpatryk) [reply src/components/user/Avatar/index.js:4]: 👍

**Review by glowacki-dev — APPROVED** [review]

**Pull request merged**